docs/ref/models/fields.txt

```diff
@@ -521,12 +521,22 @@ optional arguments:
     * For :class:`DateTimeField`: ``default=timezone.now`` - from
       :func:`django.utils.timezone.now`
 
+    .. versionchanged:: 1.10
+
+        Adding a field with an ``auto_now_add`` attribute after the
+        model has been created asks for a default value, providing
+        the default option of using the time the migration is
+        applied. The rationale is that the semantic of ``auto_now_add`` is
+        having a timestamp for the record **creation** while applying it's
+        just an update.
+
 The default form widget for this field is a
 :class:`~django.forms.TextInput`. The admin adds a JavaScript calendar,
 and a shortcut for "Today". Includes an additional ``invalid_date`` error
 message key.
 
-The options ``auto_now_add``, ``auto_now``, and ``default`` are mutually exclusive.
+The options ``auto_now_add``, ``auto_now``, and ``default`` are mutually
```

Comment:
Sure, here are the edits I had made locally: http://dpaste.com/3J1M7K1
I think it's fine to squash the commits since adding an untested feature in a commit doesn't make much sense to me.